backend/app/views/spree/admin/option_types/edit.html.erb

```diff
@@ -1,13 +1,12 @@
 <% content_for :page_title do %>
-  <%= Spree.t(:editing_resource, resource: Spree::OptionType.model_name.human) %> <span class="green">"<%= @option_type.name %>"</span>
+  <%= link_to Spree.t(:option_types), spree.admin_option_types_url %> /
+  <%= @option_type.presentation %>
```

Comment:
@damianlegawiec We should use option type name here as it can be preferred by admin.